### ISSUE TITLE
Bug initializing DiffractionPattern

### ISSUE THREAD

**Issue by tcmoore3** [glotzerlab/freud#630]
**Describe the bug**
`AttributeError` raised when initializing a `diffraction.DiffractionPattern` object.

**To Reproduce**
```
>>> import freud
>>> dp = freud.diffraction.DiffractionPattern()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "freud/diffraction.pyx", line 68, in freud.diffraction.DiffractionPattern.__init__
AttributeError: 'freud.diffraction.DiffractionPattern' object has no attribute '_k_values_orig'
>>>
```

**System configuration (please complete the following information):**
 - OS: macOS 10.13.6
 - Version of Python: 3.7
 - Version of freud: `feature/diffraction`, commit `97a824b4822`.

**Comment by tcmoore3** [comment]
It appears this bug was introduced with the latest commit, where changes were made to the DiffractionPattern class:

```
$ git diff 0ccf7ef 97a824b
...
@@ -55,12 +55,16 @@ class DiffractionPattern(_Compute):
             Width of Gaussian convolved with points, in system length units
             (Default value = 1).
     """
+    cdef int grid_size
+    cdef int output_size
+    cdef float zoom
+    cdef float peak_width
 
     def __init__(self, grid_size=512, output_size=512, zoom=4, peak_width=1):
         self.grid_size = int(grid_size)
         self.output_size = int(output_size)
-        self.zoom = zoom
-        self.peak_width = peak_width
+        self.zoom = float(zoom)
+        self.peak_width = float(peak_width)
```

When I build from that previous commit, things work as expected inside the notebook.

**Comment by bdice** [comment]
This was fixed in the final release of the feature in #596.
